_posts/2025-03-20-phased-strimzi-upgrade-example.md

```diff
@@ -0,0 +1,637 @@
+│  ├── 020-ClusterRole-strimzi-cluster-operator-role.yaml
+│  ├── 020-RoleBinding-strimzi-cluster-operator.yaml
+│  ├── 021-ClusterRole-strimzi-cluster-operator-role.yaml
+│  ├── 021-ClusterRoleBinding-strimzi-cluster-operator.yaml
+│  ├── 022-ClusterRole-strimzi-cluster-operator-role.yaml
+│  ├── 022-RoleBinding-strimzi-cluster-operator.yaml
+│  ├── 023-ClusterRole-strimzi-cluster-operator-role.yaml
+│  ├── 023-RoleBinding-strimzi-cluster-operator.yaml
+│  ├── 030-ClusterRole-strimzi-kafka-broker.yaml
+│  ├── 030-ClusterRoleBinding-strimzi-cluster-operator-kafka-broker-delegation.yaml
+│  ├── 031-ClusterRole-strimzi-entity-operator.yaml
+│  ├── 031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml
+│  ├── 033-ClusterRole-strimzi-kafka-client.yaml
+│  ├── 033-ClusterRoleBinding-strimzi-cluster-operator-kafka-client-delegation.yaml
+│  ├── 050-ConfigMap-strimzi-cluster-operator.yaml
+│  ├── 060-Deployment-strimzi-cluster-operator.yaml
+│  ├── kustomization.yaml
+├── crds
+│  ├── 04A-Crd-kafkanodepool.yaml
+│  ├── 040-Crd-kafka.yaml
+│  ├── 041-Crd-kafkaconnect.yaml
+│  ├── 042-Crd-strimzipodset.yaml
+│  ├── 043-Crd-kafkatopic.yaml
+│  ├── 044-Crd-kafkauser.yaml
+│  ├── 045-Crd-kafkamirrormaker.yaml
+│  ├── 046-Crd-kafkabridge.yaml
+│  ├── 047-Crd-kafkaconnector.yaml
+│  ├── 048-Crd-kafkamirrormaker2.yaml
+│  ├── 049-Crd-kafkarebalance.yaml
+│  └── kustomization.yaml
+└── kustomization.yaml
+```
+
+`install-files/cluster-operator-0.43.0/kustomization.yaml`
+```yaml
+resources:
+- 010-ServiceAccount-strimzi-cluster-operator.yaml
+- 020-ClusterRole-strimzi-cluster-operator-role.yaml
+- 020-RoleBinding-strimzi-cluster-operator.yaml
+- 021-ClusterRoleBinding-strimzi-cluster-operator.yaml
+- 021-ClusterRole-strimzi-cluster-operator-role.yaml
+- 022-ClusterRole-strimzi-cluster-operator-role.yaml
+- 022-RoleBinding-strimzi-cluster-operator.yaml
+- 023-ClusterRole-strimzi-cluster-operator-role.yaml
+- 023-RoleBinding-strimzi-cluster-operator.yaml
+- 030-ClusterRoleBinding-strimzi-cluster-operator-kafka-broker-delegation.yaml
+- 030-ClusterRole-strimzi-kafka-broker.yaml
+- 031-ClusterRole-strimzi-entity-operator.yaml
+- 031-RoleBinding-strimzi-cluster-operator-entity-operator-delegation.yaml
+- 033-ClusterRoleBinding-strimzi-cluster-operator-kafka-client-delegation.yaml
+- 033-ClusterRole-strimzi-kafka-client.yaml
+- 050-ConfigMap-strimzi-cluster-operator.yaml
+- 060-Deployment-strimzi-cluster-operator.yaml
+``` 
+
+`install-files/crds/kustomization.yaml`
+```yaml
+resources:
+- 040-Crd-kafka.yaml
+- 041-Crd-kafkaconnect.yaml
+- 042-Crd-strimzipodset.yaml
+- 043-Crd-kafkatopic.yaml
+- 044-Crd-kafkauser.yaml
+- 045-Crd-kafkamirrormaker.yaml
+- 046-Crd-kafkabridge.yaml
+- 047-Crd-kafkaconnector.yaml
+- 048-Crd-kafkamirrormaker2.yaml
+- 049-Crd-kafkarebalance.yaml
+- 04A-Crd-kafkanodepool.yaml
+```
+
+`install-files/crds/kustomization.yaml`
+```yaml
+resources:
+- cluster-operator-0.43.0
+- crds
+```
```

Comment:
this long list of folders + files could easily confuse readers to understand what they are looking at. Maybe a short description for each would help?

Reply:
I added more descriptive captions for each listing, hopefully that makes it clearer what each relates too.